src/Microsoft.ComponentDetection.Orchestrator/Experiments/Configs/UvLockDetectorExperiment.cs

```diff
@@ -0,0 +1,23 @@
+namespace Microsoft.ComponentDetection.Orchestrator.Experiments.Configs;
+
+using Microsoft.ComponentDetection.Contracts;
+using Microsoft.ComponentDetection.Detectors.Pip;
+using Microsoft.ComponentDetection.Detectors.Uv;
+
+/// <summary>
+/// Experiment to validate UvLockComponentDetector against PipComponentDetector.
+/// </summary>
+public class UvLockDetectorExperiment : IExperimentConfiguration
+{
+    /// <inheritdoc />
+    public string Name => "UvLockDetectorExperiment";
+
+    /// <inheritdoc />
+    public bool IsInControlGroup(IComponentDetector componentDetector) => componentDetector is PipComponentDetector;
```

Comment:
Let's compare against `PipReportComponentDetector` that is the compliant detector used when Network Isolation is enforced.